new-dti-website/src/app/products/page.tsx

```diff
@@ -29,13 +29,11 @@ export default function Page() {
           </div>
           <div className="flex flex-col justify-center w-fit gap-y-6 z-10">
             <div className="flex flex-row">
-              <div className="RealImpact text-[24px] leading-[29.05px] font-bold md:text-[40px] md:leading-[48.41px]">
-                <span className="text-neutral-400">Real</span>
-                <span className="text-white font-medium"> </span>
-                <span className="text-neutral-200 italic">impact</span>
+              <div className="RealImpact text-[24px] leading-[29.05px] font-bold md:text-[40px] md:leading-[48.41px] text-heroPrimary">
```

Comment:
Love the reutilization of the new configuration! Also just a question is there a reason that the text sizes at each breakpoint are different for each hero section? Here, we have `md:leading-[48.41px]` but for the section in `new-dti-website/src/app/sponsor/page.tsx` the **Let's collaborate** doesn't have a leading utility being used? Also having the same text-size breakpoints for each of these sections?

I was just thinking if they were all consistent we could also add these to the `tailwind.config.ts` file and so that we might not have to use decimal values for the pixels as well! 

Other than that it's starting to look so much more cohesive thanks andrew!